templates/rustdoc/topbar.html

```diff
@@ -226,7 +226,25 @@
                 </li>
             {%- endfor -%}
         </ul>
-    </li>
+    </li>{#
+    Display the features available in current build
+    #}
+    {% if 'krate' in __tera_context %}
```

Comment:
What does this do? I would prefer to use `{% if krate.features %}` or things like that instead, that will also avoid bugs like https://github.com/rust-lang/docs.rs/issues/1111.

Reply:
Also, I think this should be shown on every page that has `Platform`, there's no reason to special case the documentation page.

Reply:
The first one is done. I am not sure how to address the second one. Can you please give me a hint?